Consul/Client_GetRequests.cs

```diff
@@ -39,9 +39,19 @@ public class GetRequest<TOut> : ConsulRequest
     {
         public QueryOptions Options { get; set; }
 
-        public IEncodable Filter { get; set; }
+        public IEncodable Filter { get; }
 
-        public GetRequest(ConsulClient client, string url, QueryOptions options = null, IEncodable filter = null) : base(client, url, HttpMethod.Get)
```

Comment:
New optional parameter is source compatible but it is not a binary compatible change. Binary compatibility is important for avoiding issues with diamond dependencies.